frontend/test/metabase/scenarios/sharing/subscriptions.cy.spec.js

```diff
@@ -273,7 +273,7 @@ describe("scenarios > dashboard > subscriptions", () => {
     });
   });
 
-  describe("OSS email subscriptions", () => {
+  describe("OSS email subscriptions", { tags: "@OSS" }, () => {
```

Comment:
Is this one working with 2 OSS to parse ?

Reply:
@diogormendes not sure exactly what you mean?

Does it parse the "OSS" in "OSS email subscriptions"?